.github/workflows/build-matrix.org.yml

```diff
@@ -0,0 +1,133 @@
+name: "🌐 Build the matrix.org website"
+on: [workflow_dispatch, push, pull_request]
+jobs:
+  build-spec:
+    name: "📖 Spec"
+    runs-on: ubuntu-latest
+    container: "python:3.6"
+    steps:
+      - name: "📥 Source checkout"
+        uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - name: "📥 matrix-doc OpenAPI artifact download"
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: main.yml
+          workflow_conclusion: success
+          repo: ${{ github.repository_owner }}/matrix-doc
+          branch: ${{ github.ref }}
```

Comment:
This attemps to download the OpenAPI definition from the matrix-doc repo, from a branch with the same name as the current branch. I.e. the gh-actions branch for matrix.org tries to download the artifact from the gh-actions branch in the matrix-doc repo. When we merge this, master will reach for master.

This breaks with pull requests though, as can currently be seen where `refs/pull/1063/merge` is only relevant in this repository and not the matrix-doc one. I reckon we can change this to master *as a subsequent PR once this is merged*.

Reply:
And now that [matrix-doc moved to a `main` branch](https://github.com/matrix-org/matrix-doc/issues/3367) this is broken before it even happened.

Should I just hardcode `main`? This means it will be broken for the context of the pull request, but will work once everything is merged. For what it’s worth, there is evidence this worked in previous runs, so we’re not running blind here.